tasks/query-cmr/app/routers/router.ts

```diff
@@ -34,8 +34,8 @@ export async function doWork(workReq: QueryCmrRequest): Promise<[number, number,
   timingLogger.info('timing.query-cmr.start');
 
   const [totalGranulesSize, catalogs, newScrollId, hits] = await queryGranules(operation, scrollId, workReq.maxCmrGranules);
-  const granuleScrollingTime = new Date().getTime();
-  timingLogger.info('timing.query-cmr.query-granules-scrolling', { durationMs: granuleScrollingTime });
+  const granuleSearchTime = new Date().getTime();
+  timingLogger.info('timing.query-cmr.query-granules-search', { durationMs: granuleSearchTime });
```

Comment:
This `durationMs` is incorrect. It looks like we're missing one of the timestamps.

It should probably be:
```
  timingLogger.info('timing.query-cmr.query-granules-search', { durationMs: granuleSearchTime - queryCmrStartTime });
```

which means add this for line 35:
```
const queryCmrStartTime = newDate().getTime();
```

Reply:
Done.